python/shotgun_globals/cached_schema.py

```diff
@@ -920,3 +967,29 @@ def clear_cached_data(cls, project_id=None):
                 )
                 raise
 
+
+def _account_for_bubble_fields(sg_entity_type, field_name):
+    """Detect bubble fields and return the proper entity type and field name.
+
+    :param str sg_entity_type: The intput entity type name. If the field name
```

Comment:
Could this be added to core as well? That way, if we introduce the utility now, we can switch shotgun utils over at some point in the future. This feels like a really useful generic method that applies to any shotgun filter logic.